### DISCUSSION TITLE
App version 2024.01.23-release (225060) Flipper version 6.0.13 Troubleshooting id 4519a1fe-ae27-4cd9-bf7d-180f39854d5e

### DISCUSSION THREAD

**Discussion by Freddyplr** [community/community#103119] [New to GitHub]
App version 2024.01.23-release (225060)
Flipper version 6.0.13
Troubleshooting id 4519a1fe-ae27-4cd9-bf7d-180f39854d5e

**Comment by gil-son** [comment]
Can you give us more information?
